internal/server/server_ingest.go

```diff
@@ -27,8 +27,15 @@ func (s *Server) Ingest(ctx context.Context, request *talaria.IngestRequest) (*t
 
 	// Iterate through all of the appenders and append the blocks to them
 	for _, t := range s.tables {
+		exist := false
+		for _, each := range request.Tables {
+			if each == t.Name() {
+				exist = true
```

Comment:
Plz `break` when found one

Reply:
Could you save the `s.tables` names into a map first, so we don't need to loop them?

Reply:
we are accepting a list of tables in the ingest batch gRPC contract instead of just a single table, to preserve the fan-in nature of talaria. Instead of s.Tables, we are making request.tables into a map, so the complexity will still be o(m+n) instead of o(n*m).